README.md

```diff
@@ -14,7 +14,7 @@ I do not support piracy in any way, all the data and information in my software
 ## Current functionality
 ### General
 - Support for Web Browser (SPIDER) entrypoint for Old3DS (ver 4.1-9.2)
-- Support for System Settings (MSET) entrypoint for Old3DS (ver 4.1-4.5 and 6.0-6.4)
+- Support for System Settings (MSET) entrypoint for Old3DS (6.0-6.4)
```

Comment:
I see, makes sense then. So the reason 9.2 dg'd to 6.xMSET works is because the ROP chain for 9.2+6.x MSET *is* included? (For n3DS/2DS users, I assume?)

Either way, if someone has more documentation on what *is* supported to boot I'll update my README.md and submit another pull request.